uint8 agentId
uint8 taskId
uint8 targetId
float64[3] taskLocation
float64 taskDeadline
float64 timestamp
